pkg/otlp/model/translator/metrics_translator.go

```diff
@@ -484,6 +484,14 @@ func (t *Translator) MapMetrics(ctx context.Context, md pmetric.Metrics, consume
 
 			for k := 0; k < metricsArray.Len(); k++ {
 				md := metricsArray.At(k)
+
+				// Map metric to equivalent Datadog runtime metric if found
+				if ddruntimename, ok := runtimeMetricsMappings[md.Name()]; ok {
+					ddruntime := metricsArray.AppendEmpty()
+					md.CopyTo(ddruntime)
+					ddruntime.SetName(ddruntimename)
+				}
```

Comment:
I would write it like this:
```go
if v, ok := runtimeMetricsMappings[md.Name()]; ok {
    // duplicate runtime metrics as Datadog runtime metrics
    cp := metricsArray.AppendEmpty()
    md.CopyTo(cp)
    cp.SetName(v)
}
```
I will try to explain and link to the [dd-trace-go styleguide](https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines) for reference (it is the same as our internal one):
* `v` is very often used as "value" in a short `if` statement. To understand the concept behind this so that it makes sense I recommend reading [this section](https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines#naming-things) of the style guide.
* The same as above for all the other name changes, `cp` makes sense for many because it is also the "copy" command in the shell.
* I would place the comment inside the `if` statement always in Go. If the reader does not want to explore that code path, he can skip reading a line of text which is unrelated within that context. It also makes more sense. The words themselves from the comment now live on the same block where the action that they describe happens.

Reply:
Thanks for the feedback, that makes sense to me!